routing_model/src/scenario_tests.rs

```diff
@@ -564,6 +566,65 @@ mod dst_tests {
         );
     }
 
+    #[test]
+    fn test_parsec_expect_candidate_and_online_then_neighbour_merge() {
+        let initial_state = arrange_initial_state(
+            &initial_state_old_elders(),
+            &[
+                ParsecVote::ExpectCandidate(CANDIDATE_1).to_event(),
+                ParsecVote::Online(CANDIDATE_1).to_event(),
```

Comment:
Why Online? It should only be used for resource proofing, I don't see it would do anything.

Reply:
Yes I think this was actually just a placeholder